core/input/input.cpp

```diff
@@ -1206,22 +1206,22 @@ void Input::_get_mapped_hat_events(const JoyDeviceMapping &mapping, int p_hat, J
 	}
 }
 
-JoyButtonList Input::_get_output_button(String output) {
+JoyButton Input::_get_output_button(String output) {
 	for (int i = 0; i < JOY_BUTTON_SDL_MAX; i++) {
 		if (output == _joy_buttons[i]) {
-			return JoyButtonList(i);
+			return JoyButton(i);
 		}
 	}
-	return JoyButtonList::JOY_BUTTON_INVALID;
+	return JoyButton::JOY_BUTTON_INVALID;
 }
 
-JoyAxisList Input::_get_output_axis(String output) {
+::JoyAxis Input::_get_output_axis(String output) {
```

Comment:
It's a bit unfortunate to have a name clash with the other `JoyAxis`. Not sure if/how to solve it, it's not too bad I guess.

Reply:
We could rename `struct JoyAxis` to something else.

Reply:
Good idea, maybe `JoyAxisValue`?